### PR TITLE
ci: draw lines in honeycomb when frontend tests start and end

### PR DESCRIPTION
**Pull request by jazzdan**
We suspect these tests may be causing issues in the backend. This should help us see that more clearly.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -189,6 +203,20 @@ jobs:
         with:
           api-key: ${{ env.RECORD_REPLAY_API_KEY }}
           public: true
+  end-e2e-marker:
```
Comment (toshok) [inline .github/workflows/test.yml:206]: I think this job needs `if: always()` so it'll run even if the needed e2e tests fail.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -62,6 +62,20 @@ jobs:
...
+    steps:
+      - name: Honeycomb Start Marker for E2E Tests
+        uses: pawelros/gha-honeycomb-marker@v1.0.0
+        id: e2e_start_marker
+        with:
+          api-key: ${{secrets.HONEYCOMB_API_KEY}}
+          dataset: "backend"
+          operation: "create"
+          type: "e2e_test_start"
+          message: "E2E Tests Start - Run #${{ github.run_number }}"
```
Comment (toshok) [inline .github/workflows/test.yml:77]: probably don't want "Start" here, since it'll be a range.

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -189,6 +203,20 @@ jobs:
...
+    steps:
+      - name: Honeycomb End Marker for E2E Tests
+        uses: pawelros/gha-honeycomb-marker@v1.0.0
+        with:
+          api-key: ${{secrets.HONEYCOMB_API_KEY}}
+          operation: "update"
+          id: ${{needs.start-e2e-marker.outputs.e2e_start_marker_id}}
+          dataset: "my-dataset"
+          type: ${{needs.e2etest.result == 'success' && 'e2e_test_complete' || 'e2e_test_failed'}}
+          message: "E2E Tests Start - Run #${{ github.run_number }} - ${{needs.e2etest.result == 'success' && 'OK' || 'FAILED'}}"
```
Comment (toshok) [inline .github/workflows/test.yml:218]: same here.  🪓  "Start"?

**Review by toshok — APPROVED** [review]
lgtm.  we'll need something similar in the e2e buildkite pipeline too, right?

**Comment by jazzdan** [comment]
@toshok is this what you meant by getting rid of "Start"?

**Comment by toshok** [comment]
@jazzdan yep!

**Review by bvaughn — APPROVED** [review]
🤷🏼  I guess this seems fine

**Comment by bvaughn** [comment]
Looks like CI failed:
> Error: Honeycomb marker `id` is required for update operation type.

**Comment by jazzdan** [comment]
Yup, I'm on it. Thanks @bvaughn

**Comment by bvaughn** [comment]
@jazzdan We going to land this? :)

**Comment by jazzdan** [comment]
@bvaughn this currently isn't doing what I want. I'm hoping to get back to it today with a fresh mind. Thanks for the ping!

**Comment by bvaughn** [comment]
This seems have been abandoned.

**Comment by jazzdan** [comment]
Yup, sorry, and thanks @bvaughn!

**Pull request closed without merging**